core/src/components/action-sheet/action-sheet.ios.scss

```diff
@@ -26,7 +26,7 @@
 // ---------------------------------------------------
 
 .action-sheet-wrapper {
-  @include margin(var(--ion-safe-area-top, 0), auto, var(--ion-safe-area-bottom, 0), auto);
+  @include margin(var(--ion-safe-area-top, 0), auto, null, auto);
```

Comment:
With this PR, the bottom safe area may change how much of the action sheet content is visible, but it does not change the computed height of the action sheet wrapper. In other words, if `--height: 300px` but `--ion-safe-area-bottom: 40px` then the visible area will be `260px` even though the overall container size remains at `300px`.

This behavior aligns with how we do things on MD, but this is a change in behavior to how the iOS action sheet used to work. I'm unclear if this is what we want though The `--height` docs state that this controls the "height of the action sheet". I would assume that the "action sheet" refers to the visible area.

Is there a way we can implement this where the visible area of the action remains === `--height`? Alternatively, would you consider the current behavior wrong and the new behavior (where wrapper height includes the bottom safe area value)?

Reply:
Changes discussed on our call:

```
.action-sheet-wrapper {
  @include margin(var(--ion-safe-area-top, 0), auto, null, auto);
  @include padding(null, null, var(--ion-safe-area-bottom, 0), null);

  box-sizing: content-box;
}
```

We'd like to add a note as to why box-sizing is needed here. We should also verify that this does not have an adverse effects for descendant element sizing. We also discussed reverting the test file change for `--height`.